libraries/lib-numeric-formats/NumericConverter.cpp

```diff
@@ -70,418 +53,59 @@ static const TranslatableString BuildBeatsFormat() {
-      XO("hundredths of cents")
-   }
-   },
-   
-   {
-   /* i18n-hint: Name of display format that shows log of frequency
-    * in decades */
-   { XO("decades") },
-   {
-      /* i18n-hint: Format string for displaying log of frequency in decades.
-       * Change the decimal points for your locale. Don't change the numbers. */
-      XO("10>01000 decades|0.434294482"),   // Scale factor is 1 / ln (10)
-      /* i18n-hint: a decade is a tenfold increase of frequency */
-      XO("thousandths of decades")
-   }
-   },
-};
-
-   const BuiltinFormatString *ChooseBuiltinFormatStrings
-      (NumericConverterType type)
-   {
-      switch (type) {
-         default:
-      case NumericConverterType::TIME:
-            return TimeConverterFormats_;
-      case NumericConverterType::FREQUENCY:
-            return FrequencyConverterFormats_;
-      case NumericConverterType::BANDWIDTH:
-            return BandwidthConverterFormats_;
-      }
-   }
-
-   size_t ChooseNBuiltinFormatStrings
-      (NumericConverterType type)
-   {
-      switch (type) {
-         default:
-      case NumericConverterType::TIME:
-            return WXSIZEOF(TimeConverterFormats_);
-      case NumericConverterType::FREQUENCY:
-            return WXSIZEOF(FrequencyConverterFormats_);
-      case NumericConverterType::BANDWIDTH:
-            return WXSIZEOF(BandwidthConverterFormats_);
-      }
-   }
-}
-
 //
 // ----------------------------------------------------------------------------
 // NumericConverter Class
 // ----------------------------------------------------------------------------
 //
 NumericFormatSymbol NumericConverter::DefaultSelectionFormat()
-{ return TimeConverterFormats_[5].name; }
+{ return L"hh:mm:ss + milliseconds"; }
 NumericFormatSymbol NumericConverter::TimeAndSampleFormat()
-{ return TimeConverterFormats_[6].name; }
+{ return L"hh:mm:ss + samples"; }
 NumericFormatSymbol NumericConverter::SecondsFormat()
-{ return TimeConverterFormats_[0].name; }
+{ return L"seconds"; }
 NumericFormatSymbol NumericConverter::HoursMinsSecondsFormat()
-{ return TimeConverterFormats_[1].name; }
+{ return L"hh:mm:ss"; }
 NumericFormatSymbol NumericConverter::HundredthsFormat()
-{ return TimeConverterFormats_[3].name; }
+{ return L"hh:mm:ss + hundredths"; }
 
 NumericFormatSymbol NumericConverter::HertzFormat()
-{ return FrequencyConverterFormats_[0].name; }
+{ return L"Hz"; }
+
+NumericFormatSymbol NumericConverter::DefaultFormat(NumericConverterType type)
+{
+   switch (type)
+   {
+   case NumericConverterType::TIME:
+      return DefaultSelectionFormat();
+   case NumericConverterType::FREQUENCY:
+      return HertzFormat();
+   case NumericConverterType::BANDWIDTH:
+      return L"octaves";
+   default:
+      return DefaultSelectionFormat();
+   }
+}
 
 NumericFormatSymbol
 NumericConverter::LookupFormat(NumericConverterType type, const wxString& id)
 {
-   if (id.empty()) {
-            if (type == NumericConverterType::TIME)
```

Comment:
The commit that deletes this finally repairs the build of the commit sequence (except for the porblem building tests on mac)

Reply:
Hmm, I thought tests on mac are fixed, at least CI passes...

Reply:
I'm still marching up the commit sequence to c9f0e36d59